README.md

```diff
@@ -104,6 +104,14 @@ defmodule MyApp.Factory do
     }
   end
 
+  # dependent factory
+  def featured_article_factory do
+    %MyApp.Article{
+      article_factory |
+      featured: true,
+    }
```

Comment:
I think this syntax is a bit nicer. What do you think? `struct!(article_factory, %{featured: true})`